.github/workflows/release.yml

```diff
@@ -40,6 +40,14 @@ env:
   PULUMI_VERSION: ${{ github.event.inputs.pulumi_version || github.event.client_payload.ref }}
   # Do not depend on C library for the tests.
   CGO_ENABLED: "0"
+  # Azure credentials for the tests
+  ARM_CLIENT_ID: ${{ secrets.ARM_CLIENT_ID }}
+  ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
+  ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
+  ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
+  AWS_REGION: "us-west-2"
+
+  # TODO: pass env through docker in tests below
```

Comment:
Is this still TODO?  Looks like it's done below, or are there some missing?

Reply:
no, I forgot to remove the todo, thanks!